ihmc-interfaces/src/main/messages/ihmc_interfaces/behavior_msgs/msg/BehaviorTreeStateMessage.msg

```diff
@@ -41,6 +48,8 @@ byte[<=1000] behavior_tree_types
 # it's type.
 uint32[<=1000] behavior_tree_indices
 
+behavior_msgs/BasicNodeStateMessage[<=500] partial_data_nodes
```

Comment:
Good to have the limit at 500 since that's going to determine our max tree size.